### PR TITLE
MAINT: review QuantEcon.py page

### PR DESCRIPTION
**Pull request by mmcky**
This PR fixes old links and simplifies some of the page

### CODE REVIEW

**Comment by mmcky** [comment]
@jstac and @oyamad I have simplified and removed some old links to QuantEcon.notebooks (that we should archive).

**Comment by mmcky** [comment]
It might be nice to setup an examples gallery of some kind (perhaps linking to specific uses across our lecture series). I will open a tracking issue.

File: pages/quantecon-py.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ A high performance, open source Python code library for economics
 
 ## Installation
 
-Before installing `quantecon` we recommend you install the [Anaconda](https://www.anaconda.com/download/) Python distribution, which includes a full suite of scientific python tools. **Note:** quantecon is now only supporting Python version 3.5+. This is mainly to allow code to be written taking full advantage of new features such as using the @ symbol for matrix multiplication. Therefore please install the latest Python 3 Anaconda distribution.
+Before installing `quantecon` we recommend you install the [Anaconda](https://www.anaconda.com/download/) Python distribution, which includes a full suite of scientific python tools. **Note:** `quantecon` only supports Python version 3.5+. This is mainly to allow code to be written taking full advantage of new features such as using the `@` symbol for matrix multiplication. Therefore please install the latest Python 3 Anaconda distribution.
```
Comment (oyamad) [inline pages/quantecon-py.md:35]: Do we still have to mention version 3.5 and the `@` symbol? Maybe this "Note" is no longer necessary?

Comment (mmcky) [reply pages/quantecon-py.md:35]: I agree @oyamad I think this can now be removed

**Comment by mmcky** [comment]
thanks @oyamad for your comments. I will merge this now.

**Pull request merged**